src/vignette/util/query_options.clj

```diff
@@ -18,6 +18,9 @@
                      :replace (create-query-opt #"^true$" false)
                      :zone (create-query-opt #"\w+")})
 
+(def webp-accept-header-name "accept")
+(def webp-accept-header-value "image/webp")
```

Comment:
Would these be better in `route_helpers` where they are used, rather than here which is mostly focused on query parameters?